### PR TITLE
Adding EKS & K8s Section

### LINKED ISSUE #15: Add K8s Manifests with Key Evaluation
This is all apart of the umbrella - #3 issue

This should have a description of the high level important keys for deploying
apps on k8s.

### PR DESCRIPTION
**Pull request by christopherhein**
**Why:**

* Adding the Kubernetes module for deploying the petstorw

**This change addresses the need by:**

* closes #15

Signed-off-by: Christopher Hein <me@christopherhein.com>

### CODE REVIEW

File: modules/container-orchestration/petstore-eks-manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        app: postgres
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        labels:
+          app: postgres
+      spec:
+        containers:
+        - image: 915347744415.dkr.ecr.us-west-2.amazonaws.com/petstore_postgres:latest
```
Comment (CarmenAPuccio) [inline modules/container-orchestration/petstore-eks-manifest.yaml]: Remove your account id

File: modules/container-orchestration/petstore-eks-manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+      matchLabels:
+        app: frontend
+    template:
+      metadata:
+        labels:
+          app: frontend
+      spec:
+        initContainers:
+        - name: init-frontend
+          image: 915347744415.dkr.ecr.us-west-2.amazonaws.com/petstore_postgres:latest
```
Comment (CarmenAPuccio) [inline modules/container-orchestration/petstore-eks-manifest.yaml]: remove your account id

File: modules/container-orchestration/petstore-eks-manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+          command: ['sh', '-c',
+                    'until pg_isready -h postgres.petstore.svc -p 5432;
+                    do echo waiting for database; sleep 2; done;']
+        containers:
+        - name: frontend
+          image: 915347744415.dkr.ecr.us-west-2.amazonaws.com/petstore_frontend:latest
```
Comment (CarmenAPuccio) [inline modules/container-orchestration/petstore-eks-manifest.yaml]: and here

**Comment by christopherhein** [comment]
@CarmenAPuccio updated, thanks for the review!

**Pull request merged**